NOTICE

```diff
@@ -0,0 +1,5 @@
+Hyperledger Fabric
+Copyright [2016-2020] contributors to Hyperledger Fabric
+
+This product includes software developed at
```

Comment:
This actually does not make sense to me.  What code was actually developed at or by Hyperledger?

Reply:
I see that ASF does it this way as well.   Still seems odd to me.